### PR TITLE
Lint apis/logo directory 

### PR DESCRIPTION
**Pull request by mauriciovieira**
This PR is part of #236 endeavor.

### CODE REVIEW

**Comment by mauriciovieira** [comment]
@Digipalvelutehdas/apika-tiimi please review.

**Comment by marla-singer** [comment]
@mauriciovieira Reviewing

File: apis/logo/collection/server/permissions.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+import { Roles } from 'meteor/alanning:roles';
+
+import ApiLogo from '../';
+
 
 ApiLogo.allow({
-  insert: function(userId, file) {
-    return Roles.userIsInRole(userId, ['admin','manager']);
+  // eslint-disable-next-line no-unused-vars
+  insert (userId, file) {
```
Comment (marla-singer) [inline apis/logo/collection/server/permissions.js]: The second parameter `file` isn't used in function. Delete it and `eslint-disable` as well.

File: apis/logo/collection/server/permissions.js [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
+    return Roles.userIsInRole(userId, ['admin', 'manager']);
   },
-  remove: function(userId, file) {
-    return Roles.userIsInRole(userId, ['admin','manager']);
+  // eslint-disable-next-line no-unused-vars
+  remove (userId, file) {
+    return Roles.userIsInRole(userId, ['admin', 'manager']);
```
Comment (marla-singer) [inline apis/logo/collection/server/permissions.js]: The second parameter `file` isn't used in function. Delete it and `eslint-disable` as well.

File: apis/logo/collection/server/permissions.js [hunk 3]
```diff
@@ -1,16 +1,23 @@
...
   },
-  read: function(userId, file) {
+  // eslint-disable-next-line no-unused-vars
+  read (userId, file) {
     return true;
```
Comment (marla-singer) [inline apis/logo/collection/server/permissions.js]: The both parameters are not used in function. Delete them and `eslint-disable` as well.

File: apis/logo/collection/server/permissions.js [hunk 4]
```diff
@@ -1,16 +1,23 @@
...
   },
-  write: function(userId, file, fields) {
+  // eslint-disable-next-line no-unused-vars
+  write (userId, file, fields) {
     return true;
```
Comment (marla-singer) [inline apis/logo/collection/server/permissions.js]: All parameters are not used in function. Delete them and `eslint-disable` as well.

File: apis/logo/upload/client/resumable.js [hunk 5]
```diff
@@ -1,15 +1,22 @@
...
-  ApiLogo.resumable.on('fileAdded', function (file) {
+Meteor.startup(() => {
+  ApiLogo.resumable.on('fileAdded', (file) => {
     return ApiLogo.insert({
       _id: file.uniqueIdentifier,
       filename: file.fileName,
       contentType: file.file.type,
-    }, function (err, apiLogoFile) {
+    // eslint-disable-next-line no-unused-vars
+    }, (err, apiLogoFile) => {
```
Comment (marla-singer) [inline apis/logo/upload/client/resumable.js]: The second parameter `apiLogoFile` isn't used in the function. Delete it and `eslint-disable` rule as well.

File: apis/logo/upload/client/uploadButton/uploadButton.js [hunk 6]
```diff
@@ -1,6 +1,10 @@
-import { ApiLogo } from '/apis/logo/collection/collection';
+import { Template } from 'meteor/templating';
+import $ from 'jquery';
 
-Template.uploadApiLogoButton.onRendered(function() {
+import ApiLogo from '../../../collection';
+
+
```
Comment (marla-singer) [inline apis/logo/upload/client/uploadButton/uploadButton.js]: Delete the extra line

File: apis/logo/view/client/view.js [hunk 7]
```diff
@@ -1,17 +1,22 @@
-import { ApiLogo } from '/apis/logo/collection/collection';
-import Apis from '/apis/collection';
+import { Meteor } from 'meteor/meteor';
+import { Mongo } from 'meteor/mongo';
+import { Template } from 'meteor/templating';
+
+import ApiLogo from '../../collection';
+
```
Comment (marla-singer) [inline apis/logo/view/client/view.js]: Delete the extra line

File: apis/logo/view/client/view.js [hunk 8]
```diff
@@ -1,17 +1,22 @@
...
 
 Template.viewApiLogo.onCreated(function () {
   const instance = this;
   // Subscribe to API logo
-  instance.subscribe('allApiLogo');
+  return instance.subscribe('allApiLogo');
```
Comment (marla-singer) [inline apis/logo/view/client/view.js]: We don't use `return` for subscription in `onCreated` function. Leave it as before, delete return

File: apis/logo/collection/server/permissions.js (hunk 1 above)
Comment (mauriciovieira) [inline apis/logo/collection/server/permissions.js]: 👍

File: apis/logo/collection/server/permissions.js (hunk 2 above)
Comment (mauriciovieira) [inline apis/logo/collection/server/permissions.js]: 👍

File: apis/logo/collection/server/permissions.js (hunk 4 above)
Comment (mauriciovieira) [inline apis/logo/collection/server/permissions.js]: 👍

File: apis/logo/upload/client/resumable.js (hunk 5 above)
Comment (mauriciovieira) [inline apis/logo/upload/client/resumable.js]: 👍

File: apis/logo/upload/client/uploadButton/uploadButton.js (hunk 6 above)
Comment (mauriciovieira) [inline apis/logo/upload/client/uploadButton/uploadButton.js]: I've been leaving 1 extra lines between import and beginning of the code... But ok. I'll delete here.

We have to discuss the import conventions, I don't remember of seeing in any guidelines, like sorting and line spacing.

File: apis/logo/view/client/view.js (hunk 8 above)
Comment (mauriciovieira) [inline apis/logo/view/client/view.js]: we get an eslint check for this. I'll leave as before and you tell me if it's better to leave a `eslint-disable`, or do this.

Comment (mauriciovieira) [inline apis/logo/view/client/view.js]: nevermind :roll_eyes:  😃

**Comment by mauriciovieira** [comment]
@marla-singer please check the changes and merge

File: apis/logo/view/client/view.js (hunk 8 above)
Comment (mauriciovieira) [inline apis/logo/view/client/view.js]: @manzapanza this is a similar case.

**Pull request merged**